src/screens/Game/TimeContainer/index.tsx

```diff
@@ -33,15 +35,14 @@ function TimeContainer({ game }: Props) {
 
   const tsInfo = store.get(game) as TimeStamp
   const options: Intl.DateTimeFormatOptions = {
-    dateStyle: 'short',
-    timeStyle: 'short'
+    year: 'numeric', month: 'numeric', day: 'numeric',
+    hour: 'numeric', minute: 'numeric', second: 'numeric'
   };
   const firstPlayed = new Date(tsInfo.firstPlayed)
-  const language = storage.getItem('language')?.replace('_', '-')
-  const firstDate = new Intl.DateTimeFormat(language || 'en', options).format(firstPlayed);
+  const firstDate = new Intl.DateTimeFormat(undefined, options).format(firstPlayed);
   const lastPlayed = tsInfo.lastPlayed ? new Date(tsInfo.lastPlayed) : null
   const totalPlayed = tsInfo.totalPlayed ? convertMinsToHrsMins(tsInfo.totalPlayed) : null
-  const lastDate = new Intl.DateTimeFormat(language || 'en', options).format(lastPlayed || new Date());
+  const lastDate = new Intl.DateTimeFormat(undefined, options).format(lastPlayed || new Date());
```

Comment:
To fix #755.
This function needs to receive a locale or undefined, yes, weird. But this is JS 😅 
So before that we were using the user language to format the date but this makes no sense since you might live in a country that speaks English but the date format is not from US. So with this change we will use the system format.